.github/ISSUE_TEMPLATE/feature_requet.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Submit a feature request 
+title: ''
+labels: 'kind/feature-request'
+assignees: ''
+
+---
+
+**Is your feature request related to a use case or problem? Please describe.**
+
+
+**Describe the solution you'd like**
+
+
+**Describe alternatives/workarounds you've considered**
```

Comment:
Should we mark this and the next one as optional?  I don't want to put too many barriers in the way of people asking for features.

Reply:
SGTM!